app/routes/quotes/components/QuotePage.js

```diff
@@ -18,7 +18,7 @@ type Props = {
   approve: (number) => Promise<*>,
   unapprove: (number) => Promise<*>,
   deleteQuote: (number) => Promise<*>,
-  fetchMore: ({ approved: boolean }) => Promise<*>,
+  fetchAll: ({ query: { approved?: string } }, next?: boolean) => Promise<*>,
```

Comment:
Could you explain why you treat `approved` as a string?🤔 It makes sense in regards to the rest of the code, but is it not a bad practice?

Reply:
Mostly for simplicity and consistency with other things. But yeah, changed to boolean now, and ensured that we only pass it around as a boolean

Reply:
Blæh. That makes the "query object" => "querystring" convertion return a querystring dropping the `approved` elem when it is false. String it is....